sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/signature/AbstractKeyVaultKeyLessSignature.java

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.security.keyvault.jca.implementation.signature;
+
+import com.azure.security.keyvault.jca.KeyVaultPrivateKey;
+import com.azure.security.keyvault.jca.implementation.KeyVaultClient;
+
+import java.nio.ByteBuffer;
+import java.security.InvalidParameterException;
+import java.security.MessageDigest;
+import java.security.PublicKey;
+import java.security.PrivateKey;
+import java.security.SignatureSpi;
+import java.security.SecureRandom;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.spec.AlgorithmParameterSpec;
+import static com.azure.security.keyvault.jca.implementation.KeyVaultClient.createKeyVaultClientBySystemProperty;
+
+/**
+ * KeyVault Signature to key less sign
+ */
+public abstract class AbstractKeyVaultKeyLessSignature extends SignatureSpi {
+
+    protected KeyVaultClient keyVaultClient;
+
+    // message digest implementation we use for hashing the data
+    protected MessageDigest messageDigest;
+
+    protected String keyId;
+
+    //The default algorithm for certificate sign when the certificate is stored in Key Vault.
```

Comment:
Could you please use this style of comment?
```
    /**
     * test
     * @return the default algorithm.
     */
```
So it can be resolved by other tools like Intellij:
![image](https://user-images.githubusercontent.com/13167207/127588523-2905df9b-df7b-406a-8747-9dba07b28013.png)

It's sub class can use this comment, too:
![image](https://user-images.githubusercontent.com/13167207/127588639-660d3c2a-2b85-495b-ba51-3122a06602f2.png)